clicache/src/LocalRegion.cpp

```diff
@@ -893,7 +893,7 @@ namespace Apache
 
         _GF_MG_EXCEPTION_TRY2/* due to auto replace */
 
-			std::vector<std::shared_ptr<apache::geode::client::RegionEntry>> vc;
```

Comment:
This looks fine in the editor.... Not sure what the deal is here.

Reply:
Your editor is using TAB not SPACE.